src/clang-cpp-frontend/clang_cpp_convert.cpp

```diff
@@ -23,6 +23,7 @@ CC_DIAGNOSTIC_POP()
 #include <util/std_expr.h>
 #include <fmt/core.h>
 #include <clang-c-frontend/typecast.h>
+#include "util/c_types.h"
```

Comment:
Let's use \<angle\> brackets for includes. The "quoted" version starts looking in the source file's directory (where the header wouldn't be found).